Funcky.Test/Extensions/EnumerableExtensions/PartitionTest.cs

```diff
@@ -1,12 +1,21 @@
 using System.Linq;
 using Funcky.Extensions;
+using Funcky.Test.TestUtils;
 using Xunit;
 using static Funcky.Functional;
 
 namespace Funcky.Test.Extensions.EnumerableExtensions
 {
     public sealed class PartitionTest
     {
+        [Fact(Skip="TODO fix")]
```

Comment:
I'd mention the issue that tracks this in the skip description

Reply:
Adressed in 1d86e57 which now mentions #271